geolocation-API/getCurrentPosition_permission_deny.https.html

```diff
@@ -15,20 +15,21 @@
     onSuccess, onError, hasMethodReturned = false;
 
 t.step(function() {
-  onSuccess = t.step_func(function(pos) {
+  onSuccess = t.step_func_done(function(pos) {
```

Comment:
I take that back, [`unreached_func`](https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/resources/testharness.js?type=cs&q=unreached_func&sq=package:chromium&l=1456) can only be used for functions with no arguments.